### ISSUE TITLE
node doesn't start

### ISSUE THREAD

**Issue by robtex** [ElementsProject/lightning#1759]
### Issue and Steps to Reproduce

updated to latest master `v0.6-155-gd6048de`

node doesn't seem to start. last things in logs is:

```
2018-07-26T18:28:44.439Z lightningd(25339): testing /usr/local/libexec/c-lightning/lightning_channeld
2018-07-26T18:28:44.449Z lightningd(25339): testing /usr/local/libexec/c-lightning/lightning_closingd
2018-07-26T18:28:44.454Z lightningd(25339): testing /usr/local/libexec/c-lightning/lightning_connectd
2018-07-26T18:28:44.460Z lightningd(25339): testing /usr/local/libexec/c-lightning/lightning_gossipd
2018-07-26T18:28:44.468Z lightningd(25339): testing /usr/local/libexec/c-lightning/lightning_hsmd
2018-07-26T18:28:44.471Z lightningd(25339): testing /usr/local/libexec/c-lightning/lightning_onchaind
2018-07-26T18:28:44.474Z lightningd(25339): testing /usr/local/libexec/c-lightning/lightning_openingd
2018-07-26T18:28:44.848Z lightning_hsmd(25347): pid 25347, msgfd 14
2018-07-26T18:28:44.948Z lightning_connectd(25348): pid 25348, msgfd 17
2018-07-26T18:28:44.948Z lightning_hsmd(25347): Client: Received message 11 from client
2018-07-26T18:28:44.949Z lightning_hsmd(25347): Client: Received message 9 from client
2018-07-26T18:28:45.070Z lightning_connectd(25348): Created IPv4 listener on port 9735
2018-07-26T18:28:45.070Z lightning_connectd(25348): REPLY WIRE_CONNECTCTL_INIT_REPLY with 0 fds
2018-07-26T18:28:45.077Z lightning_gossipd(25349): pid 25349, msgfd 16
2018-07-26T18:28:46.646Z lightningd(25339): Loaded 0 HTLC signatures from DB
2018-07-26T18:28:46.657Z lightningd(25339): Loaded 0 HTLC signatures from DB
2018-07-26T18:28:46.668Z lightningd(25339): Loaded 0 HTLC signatures from DB
2018-07-26T18:28:46.673Z lightningd(25339): Loaded 0 HTLC signatures from DB
2018-07-26T18:28:46.688Z lightningd(25339): Loaded 0 HTLC signatures from DB
.
.
.
2018-07-26T18:28:47.600Z lightningd(25339): Loaded 0 HTLC signatures from DB
2018-07-26T18:28:47.608Z lightningd(25339): Loaded 0 HTLC signatures from DB
2018-07-26T18:28:47.610Z lightningd(25339): Loaded 0 HTLC signatures from DB
2018-07-26T18:28:47.612Z lightningd(25339): Loaded 0 HTLC signatures from DB
```
and then nothing, and no cpu usage.

### `getinfo` output

didn't start properly so no rpc, sorry

**Comment by cdecker** [comment]
What's the behavior of the node like besides all of those `Loaded x HTLC signatures from DB` messages? Does it exit (return to shell), or does it hang indefinitely?

We can try to bisect the point at which it hangs by stepping through the startup in `gdb`. If you'd like to help with that run `gdb lightningd/lightningd` and followed by 

```
break lightningd.c:394
r --log-level=debug --network=bitcoin [all other options you usually give lightningd]
```

Then use `n` + enter to step through the execution line by line. The breakpoint should be hit immediately after loading the HTLCs, which in your case are quite a few, hopefully reducing the number of steps before it hangs :-)

**Comment by cdecker** [comment]
@robtex any news from this node?

**Comment by robtex** [comment]
Hi. Sorry i got sidetracked.
I tried the abrove but it didn't hit the break point so i ctrl-c it.

```
#0  0x00007ffff7310260 in __read_nocancel () at ../sysdeps/unix/syscall-template.S:84
#1  0x0000000000472106 in read_all (fd=12, data=0x7fffffffd0d4, size=4) at ccan/ccan/read_write_all/read_write_all.c:28
#2  0x00000000004405a2 in wire_sync_read (ctx=0x720110, fd=12) at wire/wire_sync.c:27
#3  0x000000000040862a in get_channel_basepoints (ld=0x720140, peer_id=0xc55270, dbid=723, local_basepoints=0x7fffffffda90, local_funding_pubkey=0x7fffffffda50) at lightningd/channel.c:119
#4  0x00000000004582e4 in wallet_stmt2channel (ctx=0x720140, w=0x7241b0, stmt=0xb84798) at wallet/wallet.c:629
#5  0x00000000004586bd in wallet_channels_load_active (ctx=0x720140, w=0x7241b0) at wallet/wallet.c:705
#6  0x000000000041499d in main (argc=2, argv=0x7fffffffdf68) at lightningd/lightningd.c:377
```

**Comment by robtex** [comment]
getting new master now

**Comment by robtex** [comment]
no big difference. PoW:
```
(gdb) break lightningd.c:394
Breakpoint 1 at 0x414671: file lightningd/lightningd.c, line 394.
(gdb) r --log-level=debug
Starting program: /usr/local/bin/lightningd --log-level=debug
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
^C
Program received signal SIGINT, Interrupt.
0x00007ffff7310260 in __read_nocancel () at ../sysdeps/unix/syscall-template.S:84
84      ../sysdeps/unix/syscall-template.S: No such file or directory.
(gdb) bt
#0  0x00007ffff7310260 in __read_nocancel () at ../sysdeps/unix/syscall-template.S:84
#1  0x0000000000471351 in read_all (fd=12, data=0x7fffffffd0d4, size=4) at ccan/ccan/read_write_all/read_write_all.c:28
#2  0x000000000043fefc in wire_sync_read (ctx=0x71f118, fd=12) at wire/wire_sync.c:27
#3  0x0000000000408651 in get_channel_basepoints (ld=0x71f148, peer_id=0xc8b278, dbid=723, local_basepoints=0x7fffffffda90, local_funding_pubkey=0x7fffffffda50) at lightningd/channel.c:119
#4  0x000000000045763b in wallet_stmt2channel (ctx=0x71f148, w=0x722d88, stmt=0xbb9168) at wallet/wallet.c:629
#5  0x0000000000457a14 in wallet_channels_load_active (ctx=0x71f148, w=0x722d88) at wallet/wallet.c:705
#6  0x0000000000414546 in main (argc=2, argv=0x7fffffffdf68) at lightningd/lightningd.c:377
(gdb)
```

**Comment by robtex** [comment]
line 394:
`        if (!wallet_htlcs_reconnect(ld->wallet, &ld->htlcs_in, &ld->htlcs_out))`

line 377:
`        if (!wallet_channels_load_active(ld, ld->wallet))`

**Comment by robtex** [comment]
and last thing in log is
```
2018-08-03T19:46:23.103Z lightningd(20313): Loaded 0 HTLC signatures from DB
2018-08-03T19:46:23.104Z lightningd(20313): Loaded 0 HTLC signatures from DB
2018-08-03T19:46:23.108Z lightningd(20313): Loaded 0 HTLC signatures from DB
2018-08-03T19:46:23.116Z lightningd(20313): Loaded 0 HTLC signatures from DB
2018-08-03T19:46:23.121Z lightningd(20313): Loaded 0 HTLC signatures from DB
2018-08-03T19:46:23.127Z lightningd(20313): Loaded 0 HTLC signatures from DB
2018-08-03T19:46:23.131Z lightningd(20313): Loaded 0 HTLC signatures from DB
```

**Comment by cdecker** [comment]
Looks like we are dropping a `get_channel_basepoints` request in the HSM, but still wait for it indefinitely. This is assuming that the `Loaded HTLC signatures from DB` are not printing indefinitely, in which case it's just slow.

**Comment by robtex** [comment]
nah it scrolls quite quick and then suddenly stop and i've given it very long time...
just pulled new master hoping i was lucky, but seems to be the same problem...
```
(gdb) bt
#0  0x00007ffff7310260 in __read_nocancel () at ../sysdeps/unix/syscall-template.S:84
#1  0x00000000004712c9 in read_all (fd=12, data=0x7fffffffd0f4, size=4) at ccan/ccan/read_write_all/read_write_all.c:28
#2  0x000000000043fddc in wire_sync_read (ctx=0x71f118, fd=12) at wire/wire_sync.c:27
#3  0x000000000040862b in get_channel_basepoints (ld=0x71f148, peer_id=0xc8a8d8, dbid=723, local_basepoints=0x7fffffffdab0, local_funding_pubkey=0x7fffffffda70) at lightningd/channel.c:117
#4  0x00000000004575b3 in wallet_stmt2channel (ctx=0x71f148, w=0x722d78, stmt=0xbb9048) at wallet/wallet.c:629
#5  0x000000000045798c in wallet_channels_load_active (ctx=0x71f148, w=0x722d78) at wallet/wallet.c:705
#6  0x000000000041454e in main (argc=1, argv=0x7fffffffdf88) at lightningd/lightningd.c:382```

**Comment by cdecker** [comment]
I can't imagine how this gets stuck:

https://github.com/ElementsProject/lightning/blob/e3d95f3768f5aec4d6045b14d2da4a868b5e5648/hsmd/hsm.c#L281-L301

@rustyrussell any ideas?

**Comment by robtex** [comment]
@rustyrussell @cdecker 
Any ideas at all? I really would appreciate getting this node up and running again. Hasn't been working alright since april. ( I probably still suffer from #1434  )
Thanks

**Comment by cdecker** [comment]
If you compiled with `DEVELOPER=1` you can `lightningd` with `--dev-debugger=hsmd` and attach `gdb` as instructed:

```
2018-08-08T12:02:16.226Z lightning_hsmd(3591): pid 3591, msgfd 13
gdb -ex 'attach 3591' -ex 'p debugger_connected=1' /home/cdecker/dev/lightning/lightningd/lightning_hsmd
```

After attaching you can set a breakpoint in `handle_get_channel_basepoints` and step through invocations using `next` or `continue` once you leave the function. For example here's a session I took earlier today:

```
(gdb) break handle_get_channel_basepoints
Breakpoint 1 at 0x4032c1: file hsmd/hsm.c, line 290.
(gdb) continue
Continuing.

Breakpoint 1, handle_get_channel_basepoints (conn=0x7f5278, dc=0x7f5148) at hsmd/hsm.c:290
290             if (!fromwire_hsm_get_channel_basepoints(dc->msg_in, &peer_id, &dbid))
(gdb) list
285             u64 dbid;
286             struct secret seed;
287             struct basepoints basepoints;
288             struct pubkey funding_pubkey;
289
290             if (!fromwire_hsm_get_channel_basepoints(dc->msg_in, &peer_id, &dbid))
291                     master_badmsg(WIRE_HSM_GET_CHANNEL_BASEPOINTS, dc->msg_in);
292
293             get_channel_seed(&peer_id, dbid, &seed);
294             derive_basepoints(&seed, &funding_pubkey, &basepoints, NULL, NULL);
(gdb) n
293             get_channel_seed(&peer_id, dbid, &seed);
(gdb) 
294             derive_basepoints(&seed, &funding_pubkey, &basepoints, NULL, NULL);
(gdb) next
296             daemon_conn_send(dc,
(gdb) next
297                              take(towire_hsm_get_channel_basepoints_reply(NULL,
(gdb) next
296             daemon_conn_send(dc,
(gdb) next
300             return daemon_conn_read_next(conn, dc);
(gdb) next
handle_client (conn=0x7f5278, dc=0x7f5148) at hsmd/hsm.c:919
919     }
(gdb) continue
Continuing
```

This can be a bit cumbersome if you have many channels, alternatively we can add some more debugging prints to determine where it gets stuck.

**Comment by rustyrussell** [comment]
Or just `killall -SEGV lightning_hsmd` when it's stuck, which should at least dump a backtrace to stderr...

**Comment by rustyrussell** [comment]
I've marked this critical for the 0.6.1 milestone, too.

**Comment by rustyrussell** [comment]
If you want, you can send me your .lightning directory (which will let me steal all your funds and eliminates your privacy!) I can try to debug it here for faster resolution.  Understand if not!

My GPG key is below, it's the same as I used to sign the 0.6.0-rc2 tag (https://github.com/ElementsProject/lightning/tags) if you want to double-check:

```
-----BEGIN PGP PUBLIC KEY BLOCK-----

mQINBE6c6ZcBEADAG62jHV1wbFx/nhW3h77JZDdE0qnhBfUFCSEUG3nDzBj99Uyw
R7cJqdkwn9PsIGZKgzTlFrdcw6KfCEEJ7Jndx/gpbg0BgP5v2VqagVtIaQqnMSxV
LUiNBlfE6W9ALAsjIbzic2nhHW36XSgbloZdqXCEwnbAKTd4chY1aN4VAHcgBxmj
HQyHur3X22ETTXyX6x32GRkG+19a/ogdCnHHe2l/Ne/7r8DpoysVM6CDXjFaG/A1
UgV6jpzTdApAPWu2eTxtDv/nf7mzne2NLnvwScY8RclIz3RY35vpmNeM18Q7Q58D
XT1H9wn2gCrjOEt/RGXnXbj9E4ZQgADnAy5sWccMJC2t0NLjjE96DqzCo29RoNno
Mbm6TFHA6zNOxeJnTX55KKlw92bHo4lG3Xbp5SHH4qcDYu3BhQLhvMi/qvGjRfEA
5cvEo/UzVAq05yBaXWeqqaKCr8K3h5q6AefJ9pdyabCUzTMnAB2Eh2CIn02tdh3L
WGoQ9HcAiLZsTf2kO/dlH47o+w4R0D7IODLmVf62eiesv9tZtF1cSouLEUJjUK5l
mujIFBdQdTLq0r6EMZzsCWeNjlbVKqptFIgaDlekf14SoZA9VCPbf2I3m8WnPc5I
GEuYIpLDXnSPRi/zCoR54V7Ac4x6TEclDNb3CJDy+LSOE4XSydHy1DV3OwARAQAB
tCVSdXN0eSBSdXNzZWxsIDxydXN0eUBydXN0Y29ycC5jb20uYXU+iQI4BBMBAgAi
BQJOnOmXAhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAAKCRDZIA5s0a248Y+7
D/oCYGpe8ziWOMjI7vD5+33PKLkeiXbTOoKLxv7UWdmCtQFXfpGY7qX4lf6HIz66
59ZZLB3vFgb2zSaNZjFrM+lna8wYf1u5GMOecKy8WIX7VTl4Zq75hw6PkuIyUqHh
wptmbZGxCDltWP30A0pcDWavHftEJwK2UF79XfS3PmGyo+GpYjFeYEDvRplbqZY1
kYxITta2TQDjMWTa75hR/tBdhOPaQ4sXeAyww+OAxuHHciKE73OttPGKWEpouf1m
srpkm+c/tJENC5vGmEAF1e7IIjjNscGSHoZZqxT6UXV5zLMIe4FpLMih3SSRzypD
wSqD7DejffnJbRDbdEASc/DFdSeAg8Iumfy+pD/7xRd9Ns8Rm3+5h3p53/UztXwI
osnmDgRfWFhkWVypiUk0ucd0XxVIIKdcnPJcIf73DUbM8OpApAwi69zcD1XCcUOg
TrUpm7tsyeZ71q7efh1NkEK42s7mahgpSG840Q7dzd/Rb94oYiVEybXsHGivVOxJ
1f/nPR9D/sxqwDi82Muzz2gFlAZEKeQDWaR9E2hiFT3iDNs+E1X1kkNI1G9plezD
hPKHvcMNSZr6TPRCAoFF68+Ps9O+GPOMi8UAfX2oJddP+MzpRzx9Y4AybteP3VA4
vptBkanHoFn1co4AyI6iC2HHUUMB/9n15Gz9bc+y1zax3YhGBBARAgAGBQJOnOuu
AAoJEEhau7HlvLp1aXgAn1SlNQZfaoatXTx22Y8ZW5mG8wFkAJwMFFdpmq7JMU1F
xxF3OP3chqc+RYkCIAQTAQIACgUCTpzzFQMFAXgACgkQnkKV1gX2bOk0mxAAkK1Q
2BsCFyElZaDJYHD/yVM970LPSJpgMw/7NfHLo2rqbax99RZ/KdgonPZMVMBWLNSH
68cv80WCYWVhrMOu3vXRIWUgCK6B/o7EDloToCb42CIIc+zt6cLcLpEMnTzmwVoe
uFutwp98ipPwyDQW7WC0tW9YrYs59+PzNJzuU7kMK1fMgF7SsCrEa2M0q7V1zmlR
Srw0BKggP+/xzA/G/zjXzILNFb0iGexOU2fVHD7vXnxSt+I6MpAo443jd34vQCA1
ZI2ghgFqEep58dw19Xsdi+8nnbuGJfkFqD3caz4P44e3LEEXzvf+TpB0RTRATxWY
Rz5hElZ6IZwY8NtSvlPQfFesPGGN0JoG9J4cXELKsbL4+dAhvOkUMiB5gQlmfLj0
igLrTk1v/iVjac7x/LyyN+bV95pRKPID3RHqCZsKAjRIYT1Ot/jY8Tc7fh2f/Wt6
baSZjKxNsjxAV03Pe/8eRrCdf+TgcF+MPm+HsWOHpIaXv+U8KorIRAWq2SKnf7WN
TxLB9iFWNDLtDoMbiUvj228hVfYCZghFdYJFQmBcWVw1SHpD/tyaFVN/gtkCOMgo
Q+zsOwRx8hvUI7vjWgcqsoYPnHg88WqLmDzQ/AruaPqNNWT8FItzRLe5bFytA98f
acC7ao0awDkWyyL4F6aYDoafSX4xCWurnVBb0KmIRgQQEQIABgUCTp0YQQAKCRBo
gsrG/dpvxnhHAKCB2rIg80IfhC9LujtIgQT0VTCMWACfWSCD87ORsgoNRoECE/Ip
3ZcMuYyJARwEEAECAAYFAk6dGL0ACgkQN7cXUdx5+smWVAf/XlU0GplPSlNbFlVH
DZiRs7SWAdg2Ths+xo8mYzkoKnnYEHH0aAQLRZMkBbE4+/EnyV8N0OiJh6854CYJ
A8pyquoXXwCTjfY7aMvzZYRyWt1F/NKepYODt1TbNNWhs/DbG8VYa5rhRRaULivO
vhzsSDMNU+/MtPHb0ZYUgEFEJvwlLs65allwpchC2JGFQJELrm+90c+kLZ8/hpQI
cxegiHWOr7D6bCPR41ll011olB9g9dUBMfdVfC4ggIEIvtLEgAJLs1kfh/SvHpbW
rhEf6SwdziQ1DfGAHL+NcjHRgAdNsOZevk/7eo6GWu1L1/dLNaw6YAdumPCiSVvT
de6gMIkCHAQQAQIABgUCTp+ZywAKCRDNW4dL6sGz9UAvD/9alphd5KMiQ6Wy93dx
6T2/4KidTIp5Vmr0bevx2TXn+Cab8UuXD7i1bwaDm3E9MsgAs2DZIz8eh+G+xkfI
+ZRN3L3bbbi2+4epqtPCergTUBsZXapxN+MyXqKMvAQilhrV+wTfduE2ZpIOVGoL
eQykhtp79xNzQyz21AtyQEU2l5kNG0jwVgMkdOmX4y8VkYaXZOCT1FvWk8wNOBpN
aX46OrRo8LA95U7OOJHL/UN4KpPKqEJxaBGKqRWwor0siarhvFPJs7MXks88m8Me
q0K8HX70XnJj8eHQx8UNgdt2MhxhqzPGKDUpjXElElFDBZMfPazbgynt8kCiC4wA
kVeQcTIbHVg2JhWB1JartGLmYtSvbGbBz0a3Y199OwB6hHU7C1I3qPDawW0P0kkq
msvyj9BZS9gCAlVwjbEou9yH+39lhc+7ILpFyoyRAxfGF28Hbiq0PM7da2kh+TLW
+jHJov8q9dQ3Y31YyWf53o8YjJ01WTmFLpXS9AcXh5BoCtd3z1RjWMvxYdZTi2B8
wyvjeJf8pAn3ywqw64i+HlewYHL46ZkWsvwNpPxPu0Lg84b5svG6xZC1E4HE1D8P
YDcBlN0eojfXgfn3FkNYHQw+RXxTCHG/alhjY2H19ANI80JiQtWiAVc2gBiICtAK
PlTR7fnUFnXQrLJf3w5vE1vqx4hGBBARAgAGBQJOxHIfAAoJEATnmNRoR7NrJfcA
n3ZLeNxMjhWThcHbybJxCoTrgfPeAJ44E67LIipge4LfAaDmF5YHUrWqFYhSBDAR
AgASBQJOxHZnCx0Ad3Jvbmcga2V5AAoJEATnmNRoR7Nr9w8AnAlvBgSnlaHv/X3h
+GBweYAqAgJfAJ0cTNq9l80TnajUGvkuBcbFmglKCIkCHAQQAQIABgUCTsR1swAK
CRBzOtmEi3RMXrwvD/4lAvPCJa5twAKY+YwREUsc6ooqYnFSB7yfzlK9ZEy7Qxze
HcjBH1St9XWkju1z67MgChL341N2mCiCbkFiqbWNxn7v4uKPSfzPSSdcQA/gLArp
Bu19YQpkvyT8KUBsD0WRsgEwXg0hkP5b0fkYSWR1EheSiH30mkKKZyLTqBpGJ5Nj
urX00mSmljmoDR09erDMX8uZ/4wB3qMTmZu3kYyVUGaWCNvN8RWhPZ0xvAOuo7db
R4WFAjjZIOm75ikBv/IwUlMGlomECK3jmKgptb4HkV+e57KfI00xEk9J1rTup44r
zR7sIeU47BobkK/6/tx9N91zkbsp/DSoZ292+X3sRZK40CuFC4h9VjMdTl3UxUgE
D532YYwt54jDlPOYZFfBUaeZNFVOy2U5BbCtP0Gd1srMElDkI5HqfCZnQzKjpq8b
pWbAH45rVEgVD42A+4RKu60w8nWTmGKtJBbnhqcGFa/3fxmE5SrQT04JqK0eVxVQ
7vIPgIX1n/P9tt9nwqETaKKJSgVWt8k9g8/eChswQ9CYy2h+IQTDOAOvX99WGIrP
0sWqm6rzLMf4Fp/Av40KjQlnjaZPO1h3pnVb0TRk0tvE6KAamm1ikJTV6/Vrb7vi
sZANtRDOke6+jktx2AKo+jRxR2adT0nW9DXbcg88rIhTGlXf24AEwLaWBYmvH4kC
IAQTAQgACgUCTsRxgAMFATwACgkQSwKIlFjvkYpPhQ//RO5CV9NBQyymtxCrKBDz
HSYZUnCp5R7G7tq+5xg0EdosspBDkAUHZzf+LEUbHlLNClyRgRWzMS3uVBVLCZuW
n4AbSrjaxf7tOXWkjrf29gcR0cK8J/pNv4YgJfZwc2A+z+QNrn65e8qcZq6Bl3wt
deyaVcy1jr+d6DM0wwgHGlv12NElKFXHSZx5YnXUaTnf+y648MJ/rCEIcwaqCMLy
yVpfzM754rpg6KU7elwPISZCMkCJaIX5Jmd3yud1zSSZCqn9UGu65oPbpO/f5mX6
wWQ1knZ2Op0b0SRnz18+eVn5vVGqV24Q9N8wuGpcm/qD6Vo3caf7imKNGlYO2Q9X
slQ0tqq1k1njf9vkdQrmOcfFU0HqWP6IEYlEJXOA++LKtttw7s1/ZePekpXrSJjP
4qnrGq2cios/CtogQuluVgEtBS5/4J1p5J2q3mqiZtpD8zVz+JZHIQUrVtmUvBAv
fwHsmdrrN5KI4WBkguz9yq0M+MU73TrrQY84Zqcd6NkpDWZtQ8EydhUtLVwzK6ra
+oeJTN24lZDRgxI9QkD5/m8Pa8uq0q3/KZg4PJwtDBg74QKEAEbN1sn51JU+BigR
Om3qNhI0qvCmmv4qLyBo63xx+qIl3xHAXKrbOVPdV+bTXMaJqgO5SI4629jPF4Zz
dO61H9F0aTi/HVZ1qxxaaHaJAiAEEwEIAAoFAk7EckoDBQE8AAoJEFrSQhHAYNHI
X+AP/A5Z+VS6csg/ce+D620j8DAmWFCdxQ8iT1tqVWYs8at9QFvTA/Di41llCt1w
2UeKoORHaCiB4iYdXZcIc02yKS3ofI21rW22OR7sUrsVLlkJ0saRxXlJaKueYtsD
RUBKuqW31xObbS7CVQSFdatxAdS8sxfwEo0WX70wQD29cmjFIkqOzlxpJYLp+1KS
NqkqTS7pwYq7M9aUnBS4NbatbduFZWCTg9a3RltaMcpNbXqf/iZ0mdxXNevAe5VL
P1XBBuW2IUd4wE4+mieC5ujv+mn1dmu1HMTx5D5jU4DrCv05YivrV8eCv3xoOQQV
7eCRLHAFxo6Q4LvSYUw2jimCMrYcAXBG1tml0JqMYOlhB9byeHDOUwAnJqeOrHAw
L4Exwcjlo8Olns3Y8yDNsLRcA6KQP0XZr3xejKRMyRXG99o5I0O+jYQkYQNcYGmq
lmFSNv5sPXj0XYuQwQVAF1wLVk56zx4FkR+bUztmIfbeEcRH6DJx2Bzv1PtwvoSQ
SSUGL20WymAiCaMDsAtQKgMo73Gj+UiE3OaRmgYb3LDOWsdiAyqQFZnAQWQLlGYJ
6JFCIooSUHVPdcn991640pVNhZdtcbSOYEcE+Y9ZFlX7y/LQmg0KBgweOZ/Qa8GM
sOP8CJ8ySlmxtuzspbP0jDFCfj9fHLIIM2Lx6sXmzt9PkVOjiQIcBBABAgAGBQJO
zZ9vAAoJEL2gYIVJO6zkxocP/08zMZ0ehQPRj6L6pydyFiQziuhcTsXVIhp9cPrc
M195je9A2jjl0AAd2drHpMdLx6EDthrQCGeUbf3IU2aGafRDwaptIICOLO+MF+mj
i8zIUCfT1b4C7QJIqNpw12eBIPsazFYYmY6nuwK1G5K6IlJeLI8I4tlDMQuDi6jr
GrIx0uPt72j3wHF9oDUfsvMWQ0xjrcdu52Akc5xQ3O7r3a+bfID4YJqocGcdrlMk
jbZfoWc6QJqOpo/HPVfIpm2TrgktpRJkBrYX7UCLuMi9na9vHRAvwS71faqROr71
pOCqVZbiJg1/UIrIlSABlAgMisC3cSoaNQHPvs3CWAlSsZI6VcTcfnQSKcoNMQcW
zxyxSZFpiNn88pBwpFRdpN13zpW3oHvKdbsdd/DwZZJU3rf7MjKS7w0MZ05yzK/2
r6e5ol+VNnXou8vnj9G7JFJa+yAmUUGUY0Gljr7GCMeJPZlqjWS5NvXpQUy9xaeq
vnRFvVcieuYYWDuVPHk1rW4yKiVzIye5k/WshRYASRVdi3cxqXEgvA9n+o+qma/9
r27JAs8AyZnip3JJ73q9n7wvwojCrsl3CVApcdH6W6nREWIsVKMcAayPMoG3xdCf
paE8IheeCf8VJHiWWUZimuHAfE9NN+Uk1z3k/qNjeOXuupEn5NTD7v+eleZiT+Lp
Awm4iQIgBBABAgAKBQJPF6hoAwUBeAAKCRCQ0TuD8mSBZaCtD/4kG4LoRPEPFm1p
PAnMgWQejp6D1ovF6f/enTYhHVpOndkEyEq7PVVPxj8hemXoWYLZVGyMo4I/4sh2
sixW2gDGc5Fe2IUknnKiPS7XiQ1o/iKbBaDr5fToQ2yKtziMufE+bOMIOfaIp+kD
BYn6prZNqxD1ZWw96HV5bwe9yY5zuFLWHndySkAWVid8JTsx9XjzxSlWlJIgd/Fy
2/YozsBMHQCKDew4Y6ZAXDQ8qKdVvv87/Gp5x3UJyXvfGUYMU+t1uX+5C0ilcuFT
pSRN04OJGU/sxByIz2U3UGG/wnZmAKCVp2Mj2jrCw+dGquVpF/kL8koP/wtVfkPm
luKJayr5rt9pLN5TRlg7yGoXVkRpal5JStUCrdXOWNtpLTtiGlZALjb7JGxlHFw9
gzUbnn54rsdu/6QRvdWtUS8Ef4KmUvnX7m5xPtRoYWEUXcEYkw0TkTNMZUsHebuo
nHLEg96TS1Vndg1MQOE+D1/YkTbvexsaMWpbKSfAQkD/h0Q73Ij3bpMz2atOiHm4
F4G22QjO79+Z9IDDIe7O9g+2oDC4Gdq92PFQyQ/nSnDcjvCIaAZ+tj5MM7aFIq1h
PPmNJ11PSzVTMX7QB0w4QXdi0xiMwZjti/xnIgQQXQW1bXIai7ykiBbodiXPoO96
Lj+57x3ilvqK4XVuCVvAb8EmiIsBh4kCHAQQAQoABgUCTygncQAKCRA6k2GWwJXZ
QfA5D/9hwnChj0rdLGQv9o59fV9QNdHtEWQd0B2piAfX2z9hAq1lJqiVKN2emU85
7JOnFOQZEDtgIm70FW5lQTGIYVRl3YoqF+FjQwBMEFJluSiMSndMRA/bc3SasUs3
teJyIQYeEcZPcXJ70lauKfuAkIhrXH4g/G7nsiJRF8KxR8Skwc/iGKjhQ0mUM0uX
tPA6Rp/iY4E94M4ef1ofvm/GmKgMkWQRctVK8ZX60h8LUuohtOclMeZLJdRNAooD
vq7UCL+2K4KTbw9y3OWuKBJ9Mr5AKR0NrRoQ50n0cSefJyxHZP0ST6xiHwdno5LL
41whjUX2Fe9uIWOecktTd8j1LSJF8aFNfmi22LgGOXRZbMzdUBq2HTkfkQx7W+NM
Qi09bRx3/Q0inaJ/qO9oT2cQXnJKPzyYjcgA0ggjArACjP+1KYCmXBqLGiGfP620
KmSrEegpuQWI0Z/O2cZGhSUw/ASroOwQPtZq4jS7Eyb+LnjxPEVUlcNNu8ZA/VdD
A3UJtPsdfQL1/z+QvXk3auwm81oygDncZj5A6t2V1lYnkitjWPfBfXwdGNJ+mbfW
lWG05hOzKe3Zsob9WCcuqk4PMAZxSk2g1maqkoYQxcFGKL4iILgoW2QeCdYkx0dF
odvvf0kgrTr56EgyJ18GaIEpxUKw1s3jFF2ESPKJKcU3bkHKNYkCHAQQAQIABgUC
TzLt7gAKCRBgq0f/yQlSJycMD/9/biHhGDrCynae3hA2TryIa1+EKhXsK2c4ay4o
paArww5HI8HVujw2ZvhaEfE3RjE5w2WfrnvswwiDzW/giTpcJEBRSfIzHsrGcTrb
ZKWZ+JvPd54Id9IIO1SA/3WqvRF30O+jwhr4nEWmZQiYV7v9duDuquFf/FqN/H3G
sB5+Stzvr2G/DPfOWO6xONFQ6apkPuCJi3Z6xUe+bKGeTsyeI0DgQS1R+2DXnft8
4yV9CK8l2KtNaAHPq9X2E0JQI3KL1IOWI6xsqXTEvKsrog0ghG/PKT3REAWVRFs/
qQFjQXVldfouheG1Vbbu2lzPOQY+OrQVC8bUoN9FLG+H7BE+GQQnigU4tZaVQXfd
LbB2xQp2DivLuQrvVCtz1Bp/P7uXQCjDTt1/mQn9uNuB9vve1RF2FQCtHbCGfp8y
hY2b5b5KsVvGtoPgKGFcvUW4wE4ynoVmyz/2f/+BnP9sH5YkXvWlhqRc1zYFWrx/
v7Ey64+JTz9Ys3+EUm9GOeMV9I8PRvlpB3SojBMAHIJWRGo+nQws+0nWKm+N2Z1v
zsLZtQWX5AfkND0c5cXWUeReGqOqQzg6rSe+JhboxIRPxquH59qQZw+H9ZURCcFS
H5MOdEkW8JFPDq/a3hIxWpmv+dWjoBciMntrxtZpF2oAN5gvdQObagIaxivFskhg
jERMb4kCHAQQAQIABgUCT0SJMwAKCRAQF2fsuZRGSCV1D/9PhInF72hbXOgdcmit
uM7ciNiLk82WXt6JRYnKgdWi2Vm6566Htnpcrj4vqCxmm3wjZrNf3WAPRRV4lYKm
ndPFTm4p4f7qwzusx25lyAFDvm73Mnvt9Rn+8CGIvBFhoMTyg8AkOhnnDlLtt24L
lrhGqj0OJ+X3tvCkrU1DxIllZjOjbK5h53rmtcTbpEawkhN121fZiVRdQJSB2fou
aaLhFoVGBYXvaQpx0CNZKHv02VHyqlSb2tMlBfjNIf1OIK51XqdGZy0VMxdIWXuG
P2iandsWrGVV/9JXLJJ93LfbNJltvIBqajOxN3nXQI2MjS1A395H4/ldKkEUioh7
JR3+ECp55wD8ILq3ZFVwm+PhVhDIpQA+T4mck5NcgWSISVPtR03cHg0+D0jNuKoN
2u2j4XBi3WzFNyVEtUaBVy13Ner4xWpVJQAec2TBQKOQ9cK4iCEz3MyiojHDJ51E
JK3g76e6/76WiEFh8TEXFWX6ZVQdQPprwvHRCcDVQldPcmdxfiIG3cusJdJUVhRa
fqkI8apABaxsGDBrkXMXa/+wgSpwmtEY6+Iegk3c9WvP6ldMYGdeSVECKL7esz4D
RAgltyrYMfU1N2jRjYdzqmkCh7Lk5zcaM0feUYU9BMDncxFd34mRTqzhe4rj4S5M
+nBzltWOQzoFYG4hp4GWd7jmeIkCHAQQAQIABgUCUCG+kAAKCRCGEz3YK8EguS5U
EACHzsVMHBQrWExphYEuA8cyY9REEcKJs6fSg/JokXuUjEhlUPgrZY2wlVmx1JxQ
nZBP6CkQEX1Xnow/RBL7bPz68YqqAjtCTyZIzdim0nEl+UGjrSQBWw4SS2U0jtfy
SI8/90WN/ExAZ3s+zbD7g5DxnT3VE7jKmgn1E8JZ2/oxpWj2T/ksPi6rq2pD9PpE
dRF1GUb1F484wtsY8fTsMkLP0dLH2eq76OdStgBjjfoe9CPFf+aFp8/eNKK04iEE
zmXcLdPLvlPzjf3mftnqUn2jE6v3MbbivNutF7YudlwZiVqcvluQVPAikkRNpjg+
sDsQLZV2TdboIs+/k4Wdq2ahjjClLgdjiTreo6DDFmTd6nqUV+4hH2Ye4ROvnMdt
B+nnXIIGI5Su2ZgQsfwO6OKFjEB6wNX0rGsjOy98CaBp1sM/KYxRgn8QPhgdlrdh
bKVro8H1quZzsvQS6rzepfxVPqZGB8j9xmQYamVjKw6xFHY77Sk+PVDGpeqSBWCd
ok/dqZERC3FNC97NO+Bp8Egq1gXz4PyFxkfjTginbZbGOgsDWsqfGAi4L9qSHWyv
T2Xe8n04yGUmzGXkG+ziqH3HbfmCfh1+WsFzXZDfYd6gGLcc9xk3LzRuvoFZKuO2
7BIpnn9r1A4wXnTNfOVYXvH9gP4Ge008tybz33rFH0jODIkCHAQQAQIABgUCUD0N
RAAKCRCevxLzctn7jOBDEACK5YDAyvsECb0XBS/2OYHzf8OROM9xyoqryNjybVSJ
3kOGVdBuAn5hTJa5nb4PmKgJWJ20KM5ufa0o514eDwQ1Li4a6vrkNJXutIChwkE6
UpyXOgOYcGsifJrvzJGtDJfOTUFYSHm1hho55sV8rxiy8KSbo6m8EGwQ2HH8LrtS
LmG+M5GfA3NlIlidX51+7Z33q4c/N0WiYcVqqVj2gdMWRs0Ue8RkhhRSmKIYVjYe
gXf6M1QoxtbVjGoJAyE6ERm1AMvxZziuSESkif98Q+gT2gJKxDgAcbnV0Dd1VEop
+5f5jNRVv/rNc16QdmOw4xBQkDMPK+q0VJxpCbbipdC6B5VZSUlh9k+cjuFqmOM9
XMDVJdhG7W9hHxDYUsjqM8FC6zColc4P44BwzqY30pFxF09BU+cMUL9I0uOimZ8a
6sovnNgSgC9iQKBDxuT1O1hFJWIBgG5TcKe9MNP+eC+Hvx3a6axLDPFMQhAeDjgW
B/2x7ChZzn9KK/sT3pLqzy9NsAX8Lj/QUAK7PD9iDPLxd0x5ZsYAY8zBg3lzPnPZ
tAn1EZgzv2I0cjd7d1mlbwN84jx6ozrNUo3bY1RXoK+eAq562bn4jPr18qKVIjXg
6nrYLV7vNu8G7OE39URTix5dx8chaXJbmWiDmRhOhDrZrcRh06sv7aNmUkqM6CU2
oYhGBBARAgAGBQJQYp2QAAoJEM+APFsotDa7FPAAn3s+2UEc0Fj+J89eOe6pY7qn
6pI7AJ9TU34XQHiB9GKZLVn/jDOkqm799YkCHAQQAQIABgUCUGKeFwAKCRD+AB1E
yAIYZREbD/0W2Ph3rEct2azqZQko3ocgPOlJskpWkj1kX0UEQh6z7gMenJv3DXCs
qGhjsqbPeubVAVjQJPtZMIHEUP1/mTbvxeXOVm27kEbcHklySP5He4S0uCHXBKl4
DcB7ZSS5ShA+SV6AfISxO9KoJ5zCde59bV2pyOxQNPg6annQyIJoZNWjzoLydGTk
r2NuEl5finTYo7AEWR79mnvDSyVks0kigMTt69SHHB26Oly/TEJvd1i56TwM6dPM
VZFWu0tsWI/e6FuK1/UAQRdN2z0On7V6VSoUr+kQmzje28ZTqKzRLWbtmOSuRxHU
Y2NtwY8Wqf1Pzw0JnDYUpzWn4UMAYWnuyypvfBYy7zhagBwTm0kIP9T7KyYzaUq3
t+FMr0ERtBYvQCXx8V9XODxYkdWGDLnh92SIImXzwrMJqpG/ChDSnH388/tcq+S1
MwyqtXJJsITB5ZJXwwfxEQy+I6goMmENlfEiQ3B8WmAZ0GTe5KYDmS1CsAHheb3h
FP+HcD5D2i0NWIbRBYlALsdF3H7KlicSZx+G8ncq3f8KvRbIUqIfAFW32ulrq40t
UN8yutIQI/wwVT+8bc6J7IKfAbsEGNr20jgLCd93DQS214tFD78a6p8sc/fIKWFG
s1e54lD8T1+63tvmEHbkcHl3ON3l9z62ezTzpnZWv4k+M2BqBhV/QIkBHAQQAQIA
BgUCUp/teQAKCRBLaTos7enXclBQCACHEu9q0PivnxT8EVBwB7qCt898MImTgqXb
5njCFPkLQFGxn10tP0s3TkBw72IcmAf31IS73xI/WW2H678IIJWqkXXJlN0oFg52
daKD3rDvWgbFXDf39iL0WQBisP7EGSHtawr7KzIedI9y8ISgh/usih81O0RNmhQy
Y5rTMmuOEd8wtYqAZBmjy86dDEiDJxDQBs+FFF6Ja7ykDazIh6N+XJ9H07ZCqEWE
mbeKlHptgss/3rPJ/sAkYyJhuqkD7jmaF769KwLEew+A3mwvDFyVXBp4/EsLrJ76
ogRv/vaZN8Qpv3ShJoyKEJje0tazELjUiJJrp4V59HSUNEZLvyneiQIcBBABCgAG
BQJSzUP4AAoJEHxWrP6UeJfYmQQQAIBXZRnsiuH8Mu9/4mRLugU54kZs0Js0069M
hdZmPW9FGWqS6H2uNDre81BREqhFFYPvZRRNsO4o9fBAnNMRDHoL7zjRfUdCYLft
/0Uh+EYvDmyk/iq2OeKwHH8EV4oQDvVtBpdXbnxDEs75sU+FdWjp//eqeAnrF/4l
VJctLa588RTJ8mbScu3X+Asp6nsbgIYrMEGF22SdV4OXsYJR5lehct2GjjwotlLC
rs3c1n9HCk3kgf7Kkx9azgVEqTdnRcVr5jm9UwNtZStREq/jfan5dP6z75mWgPSU
d4V2AWljoxrvPTVl50Y/V8yHTe/pSwtW++C6Td0QgFID3l/q765L34uHkcaFvIha
c4lo3tMj3K86Gu/VT1O9YRe4ZMdTDCjb4IFmxxvp1a+V2nFsk27DZX/RVT30k/uo
acWzqdhg4TOMhx+wrNPDyZBZgAzGH88ElfhhuVR3ON8roy1YWkkWCIPtNL8istyf
WpY0b3no7P7eoHSJStvxdrdIB/F+E53FssuGljojD5WIHh0AdcTSg5oEM62ji66p
GRjnORtrGXG5yGQr+FjgZTBg4OKCakXT+UdNMthwL4Oz4Oin59NAAeuYJwC6cnmb
ugaWv1IP8LLv5UzLcJ3rAdh6loLZ4oCNHQO3OwBFSOsmXFE0noXONC9riI5kAXxk
DRosRS1FiQIcBBABCgAGBQJSzUQbAAoJEANIAsespDUKDaQP/RCFYZ8NOYyCmFmm
xVaBhYBv9voRXz/DtDjbVDx7i0fuv6lxXU8M2dht7SAYF9/IppodZuhjSwKwWfL4
EuGEF/YiFGv4gIl+CxnJkUSTVAj1jVGNnT5i+w8FE7cV0MTLx3hn4SwqDARS83PD
6RhBBaS6URWuRpUK+Q6VbF3tOYj2bbwHpg294K0Tg52ITlYyCBzjVNFKZGA3Jy1W
SvPRiHaz7KD/EPpYct1ET516EFhIxdodgEAV45NdcQ3/WmgJql8h/RCviwRwTZFM
qyYyGKFOzA+nrHQy/d3Ke63isi10aEdVwjUagaLMDKs2ru0T2j9rocX3yOvO1nn5
SRcVFcOAdT+XURaJXQrynxidFjbPsHelFVigMZMlEnPxEGr0/ozdb2Wc+7mnQaWS
n3m+cK6WQeDX1QZpXTveD6OFwn9e2lFW6u6/oRHtlEJL6/SGa1sBaHi12qhgVHlJ
N64KrzQ0YkaoN5XXm/39wlthqamNaEYH5r88TWdl53/5P9rVh0tQJ2Vp56Lw8xYl
rET2CJ1Pwm6jLDZkqxTGGa4pPouTW94Lb9mm5qkON06VUI6w7RtbhrXvhGVNChp0
Z8DLiwGfh5ULteQuCBTEDZAD6z9Y4sKB+WnODtSOMZI7lzU1/6xtSiiOtJ43/f7B
mk5KEeHEfygqEEEUYj3eWSAhCKVpiQIiBBABCgAMBQJTmngGBYMHhh+AAAoJEBTq
+TfI+8kb/7AQAIjjhLSqWosKl6Cpo9xwVqfIObVnnPeXkwiz/UJdNcrPh+aQKLfh
pXoqwkUw+kEJfxoh2DQ5dsw/vm/yNv1ctZlzR+AkWZCK/nsrKX4dr+5oZe2/vJ+E
hRIZoiHMsaZhy5Ao7Gmb/sqG1qPhg8or0Y59ovDt9ATZilBf4U+PfB/VJyWWbjQb
EkailKeY3FV8H6xCdf0wqDK128SDtytWJB1uUP5pHOoTz69wX+DSy5cJox1F7Ojc
N3QthDujjLToxEkTaspcQyae2b90bqJB1E3nk0siYe3a25fvK8wXSJ+IwWpkOWDx
AvOT8Vgo+peCP/KaT7uz2gGrz8W4xBTPXEGNJg9ecwte3x1vvZdJRk4L/1u4mq+2
0nvpQEveuZTjRxcNGmWoa4lGmmtZUU0GJIoQ6tAoj17wtPM9yiD7+QsAgV1uW06D
Pqi67fVqjCEaY6I3bQzIq3qmq2Syxfd4YdzpeK5taXSzx2nECAA2suJ7jxwR46aL
SmynA/wx3e/EBv23uPRihCiNTkHzpp6cYIY+WBkcK61OueWRzJeXeVEMPNzgZ/Dy
h/HqMmuklW2GFMbkEhxrgDZbs/7iNhn8HWxDYIvxPTEXun5GpHBQotOAZMB02b/I
5wtBUONRGyTq6YPV1YKpJOhnBXfWjuQGo0u3HWOFL2wPHZ/gcpWjvNkJiQIcBBAB
AgAGBQJTmni9AAoJEGw4ysog2bOSWJwP+wctmyhED5gROln471Y+UJYHaXLD22Aw
p520YxIaCInLPVpqj/8O3WV7lzAvxKijGvArV3E+cK8uBANQNScQaD7Qup9WY3h8
2AL8l+6wkHAzMAzjwmyBdM9vgVNLTWb6/NM0nFiRe8uJpQbKcuLVEowCm49bypFp
Ei0aPH6a97sciMS2nfvLjCoXqtpKdbYj2a51XMTSWDh3mtu4443yd83gCovc0r9m
aOhHzuZQfGsKVqzZSCVehHHBuMSdOGQ3iSGR5kguzbPTEJkpJeRSq/EVUOBMvzal
wZw8H+w4s2Ywsi+emJ0yZF1MgM2xjIjGvIX1sH7P5vwgUkKdm5MoLilw5AL+zjM7
MBVXbqsjkew0gbyKfaGfTf9uObr4+nmpi2Wv+HjIMUntN7btOJ1/0LuE3vc4Zv0i
EAINNlnFc/iVjmjSJoBsAkgEskccsR+NTDMPNAruIDJBnWT+hoWBhRFzNje+alGg
SNE8UzAcR666MA8ndJlbK0NQOYavWxL01njnFgFjFAT7yMgsZNQylr99Z4QYafiu
+l3NrznkAkHRPO0BZxGa0fv6hIWkpyGLTbECE6rHT+uVHAX3AVE2c7iJN44exkdV
Pmh8hBsA1mR33Memg8VSTvtNu5zjY7PZwlfbOWhMx0m0+/NsAm3M5zq45UPVrHSt
DviYQcN/gt/tiQIcBBABAgAGBQJTmovzAAoJEFcNXmq0LVJTFxQP/A7ZcLvZWfn9
WXALzToOkRqYG56ZQHR3J58xEjq2jE2gqqxyK3jgZZhU/Z3L2eVJroeKchrecS5q
RgYiK6jxHe5/1Ks27ip/HMA+/M794S9ROZTE2s21RyG/DHQqe2KM53gpIPosspF7
tVLx2vqbqsETodphsfp1VDkl1QGC5D9vRuJUtr/au977fWGmth8tFOqSaDvCVvCR
/pIXdBPokHbPWqy9diiSs0sGIhIAn5JI2nClZJNIWm9O2reKnJjC6BrZKvoNzcei
zt1S/uQAkOqqh2E4B4Phl4cG9Mo1LTr6+BbqLH4e5ant+8bOTjbJSx+K4G5abICL
9avTV+KH6rI8nkEhJlAwON6rbc2wds/xiCJlzz/vdhfo08Si+/4ZQld8vNzas58I
kfaLB880Os1X6RuwsUOdTke2ev14/BqVngiHnmkLGrlODWVqwDlGe4Ng045rnqyy
GuzVCiF3dyiL7yaYtlgRG+Ia7hC1s/VU8TN9MbL/KQDAAUYDnn+hD78vSotVtH22
hJKDOqSDxlCom9bw97kJYJE/j234q07dT5jewqtqkSvnqR3ejT8M6nMkPm2EEq3S
PiCSkrqD69+CTvchpIixe/8h+F8wKt7NA3XI5dcfN8/ZH67iI5APP0yNVeoKpkxI
7h2OQtjvWmj/wH5OOu6JjW9+/R9dcCgOiQIcBBABCAAGBQJTxIPzAAoJEOZqTimv
57y992kQAKfq2+bM2Vy/zf4X/+uQwBY5PWLjijHbo8PLgZww+kgl2qHVQa+MK9Qq
bL53Io6u0xR7wkE5/gnVr7Gk8rPRTRFSuYeQPA2v+2D81UxBgu6ZU6XJWhKsXV3O
5eUs9Pk0peO8MCG0C5skjxeSlbJ3MVug1Lry22WTquESH4PfIH99PbDCTepT1+kc
8q8XOc/J3mOJNejM4bgV0xrN69HEgtILGvGYfc0PsADHzww9nYD5uo3vxNwiUVU0
nRPQ+3iHqwVgVnLdqt3uuaOxGJOF3jBLGEpSL+oKIpcgoSPqvmGiSuruaXKMtH3I
x8MTCfo6S9yJPAAbSGA68Tulr07PMnkv+7m2Kz/OnCEn2tswL4i1PljykxxzffNG
zAo1vYzlc2NbLduQWbrwxdo8M4Av51TMUfyh5vhhx94YciwthIWkcbwvrf/GvN08
ruOs/MhJqgrwmRgLQKrr2L5elJQYZ3B7nAVd+A2XGc4scl/kkCiEWqJDHr/VCzPY
XskpVLHjEQIQxOHl915Xeok3FQQUa5BtdqRp22zwA8Q5F9N1kKMXRSQjZ0cavCgk
jsUj2cuqp66e7ElqJmMPyeg7k9oxG7bB8rSfCflucC2ZkIZ386OL2gSiLXA3D91v
izuvbLDMUAvQKOYP3xqYCqQSG5H55AHT7jRCv+hCZWWkT3BZJaBPiQEcBBMBCAAG
BQJV9xHoAAoJEE90TaH18VOBmKQIAMeU+ZIzfZ4royL4duyIJt8Ce0E7dWEe0y0T
TpDY5MpqR3yU1Kdddz1cfux+pLs38ZS3JhdNrSkkd6myWP57NMczVVAQr0GYHtoE
wRcPEX4hzx9Zgpk9HWeq67NuF+q+oJO9MlTnc2Upff4Ax398JPFj8eu5cXEKe+kW
UvxfV4JnETt/vn8O3r8f9g2pjgvPUHbPdKnK8ZD3XgkuBIwqqrPinmwXed89JhCh
LCYIvGFZPuAxdTbuxJ2LPPZUAk33r0t9wsTLGnZNBdlOnG3LxqU0y6HbEg8mQ3QT
jyB+3QZLwQyCRvJxNtc1/2i+wgO9Li6L/2t0ha+AMXB0XWAhexqJAhwEEAEIAAYF
AlX9noMACgkQibu4Zj4uZc57FQ//bvAGkH+CFQ0DUv3Ow5C60xY83Ht/aNsDmwTy
CdWBwB/ouUeOsUKunhjfEqxYmrjVzOdSlfTaQmIk37ezDXOC6CEoKcGarp2L3a5O
kJzVE4g3QMrrgIb/7NJv1Da1tc7D0j7CTAiU1rn5gr/1at8beRrYYqd/F7MOYyXm
Yf3N5+EIN9TBTbn9fq4d9DpEKJAhWU3I9/K+IFcWBW+5mMb+ZmPPs4k8vUU/AsiR
mqXCKDshcfbvYnxJ6RTMAuRoqCOrtJA3ReSOib4QADNEGBv0MGPEbgqV1f2lVlP7
Nxbybew6hrY9MsYvRD7lZAdDqdintRuW8+YH0hN/5MuppztAys8tSrdNXbCHJRnU
Z9lJbvMn9GOM7kTKSKvT9AuNo05jdZcQWAMjoGbeVIr+XYJyhNkYFHTwmmWZ2yUJ
5vZ2b/ProJ+qATC9ffYTWwy6KdqU5pp7KgWs6ysFbgV9FjPofVHWYUU5RybW/sbX
3dj/HyJgs9QJ9bc+01nbDpDIc4bx4hPvKNNu7eTTuC4YTdFspniKQc2CU0SGCHSU
mJWJ7yLu8xEOdiDuDn8pYQHjdmXICPe39WsvxBfiEBKXYkmuGpE/x7d08bLS41UD
ssDKu88zBTdZWVGj/UES7QC00gSYd4FiEiwMHfNXEf1WGJe/0MzwBU9bjXprfEUD
wgd6AoaJARwEEAEIAAYFAlX6/d0ACgkQetCpHEC9AJFjVQf/VTftCIv6Vpyz2Taw
NQ4yM2c34DbK3acXvQwx8E5ed61UoVhOTYiK/gF9kPqnXh/f5P8ktmI5GUx19z1z
2D1kq0ltpOL7bW8MOLFdDytHQkzKtqulxIkQj89j5XVPVXXR2dEL0U9vMcPCWc5C
7yLNyKCV/u7aoNcbGsWP+BWruv+DF1oS6ul9DdTMyhdxKbM1PS7IiZnyfjTYm2gG
nNF5xPJ0/4XTelAbmN1MNftLZxymUybT2Bo4+F01LLSKq2GE/Lj49UvY9z76c/n8
aK/ust5GH3v3JgHQXoftXg/uc+VQKpO4P6HE9vm9Ryd3xfM8NCdXVksG+ZFHYPE5
4F2lpohGBBMRCAAGBQJV/cWOAAoJEKyFk2KwQTv6WfAAoI9jz3XXOwC4RupJRaRM
7VPzgqs7AKC3GBmhnUDSsgX9yd9xR76NCzi/1YkCHAQQAQIABgUCVmfRMAAKCRCG
D+uATmaTICkhEACmiQqVnTdxlUlWU39DtJQrarhymyq4HhTVJvnKsWnAR4/sqkAe
THxV2ln2cE9lxajiRljHXGWg6feCvNG5Ya3uZCJkBwKSlxSp8Lp9iFnnYxkPUOta
Jk4BgeHRgX2h5AtVmIRwA7zp5veAsTCzxmJubJjXaBdwxUukrVPMh05iGGc+NVvj
GUhze4o8UZDc9j1w4IIAt5xlNePg6ILATWIaalDTdFfEUH0c1HBI/wXM0wMdoFFa
h9O8bdRfsyK/ZC6FNZaxeije3jtGtEWqQkmI7QZE4olJPXclq0/GTzfKqtrlROJz
9OHGVBZSoodVY3sXg+p0baHXTIhZspdJqvN4qJ8OKTTs3MpQuSEuHOZMtWJReKDt
QjLKuli9FHUtkmL61LQtEYfLwszL/Ska/8XFIEKZt8a2Jw01LeTQBdl8uqHX2ZcF
Cwc8FeOailnSlqRw9PTG4Z4NkQbhL5qnkfGY6HGE7nEc7FMyIGw4qayK/3+47o/T
DMAcAU1KhcRX6WAgYJFw1aoy6ZNnKeMhgeZg3vmATYuqcZ90fEjunFgQIxuPiKXz
29JNB2RyNIvKCc1Bw19Pb6OaPLJXoNIaA80NjK6SigXNPkNOsznqwPOsFr72gBKg
YtWYxmrwNMqwZq4eoOBsqbjJ2fAQZfvtkKj49AchmXKyr2EgdF5Afv4HqIheBBAR
CAAGBQJWZlFdAAoJEH19Eb9inVpnjjUA/3fCdSVB/ns2JedcXO0n7kMXBgNpg3K7
AyVD5axEVRCAAQCzE6gN/4PXExarW7tSuqr/N4ozSFoYX9episxbKG3nAIkBHAQT
AQoABgUCVme2wQAKCRAXq4vDILFR2ELxB/9BlpzKAHUb0C+651Gx6miecD8XJv6D
d7dlI23CXzEmwPmJ/uYYCnl55CXIKuP2Mq9nboCrVytVnfvquwkql+Cg0btdoH6F
aL13Izwtx8PK7FMfycLKTisZ1p6fde1lTnUlMx/CVaIddVa51JexSY5x9zypR9rR
2DmCVuZ+jCewcndTP2MfbShzZS3lIaMrWv6GIZBDzbaoEbG80j/ioThgIAEyktAV
+cJORH87T7qp82aZBC4lB+xMbOJZ2J+Uh1vc3tkJa0gVb47Cw/EMp/qQL0awFZ5a
YllSkjbhTN4L9fVL5mj5sv9+6ysIm2t2BusrNfvxsMiHSZkHXkXU7Dz1iQIcBBAB
AgAGBQJXcIfBAAoJEGt2WQeBR3CewjMP/1+2hS1X1SWnoBJZmplDkGBBeVUEHpEb
M68LU55vc91pj3psn+ZlXgohI68xmBigPlwjChYlbvWl5DX/UUZYrqxdmtc99a/B
e7H7jftgaOwxeruw8Wfea6Z+NngKsKh+VQszd592Q1zKy2PYNctlSHtn24I6SgiU
EWvYa2goyktYP5mCFV1zXDF7davyJeI3Jd5SaEndxn7qsPd1aRvaOLlP1/Ba2HnL
PNtBrnDjLFcFVFOxmCxkKuYGbRJQ7b4yT/sp9L7kwNPkYtMWck+233SRGlhlzI/3
2BpgIUHaKTmxo4wdjGuBQfKs3F6wEuNDuP+bqXgBmlbD12wIsjamNUMHDd7oT8Yo
0GULmmdBwcFM7ACpmz07RrIEYslomLWWq70LEbuO+NKGX4r7cPtaZ5OdMIJofUFK
8BJvjTGK3yvxUFcRqyS1DBvWGQChFyBb9ywwRjmNTapc0iqKspQbLvfNKIqxVT+f
kX9GsXYquVPVDefBK9m24hTthhdZqCnncpbvAv6HHtG9w53D87F8mGWPX58479Gu
Fr9YnckpmaMsNkugQZ5ALr/ZqwPi6VTfd7BOEL/YtGtZo2I1tE4KjTJ/CwiHR3vq
dDwSqOfxvw5xOr/XkZzFG3VdJyklYr1fQ2FtzEQXCXMVpBsTvxHxp2HLzNB+UPox
WvsBR9NAZgfZiQIcBBABCgAGBQJXctyiAAoJEKJtbZ/giO1YfR8P/2VJz2L+/iic
HJ01RIl5FDNdLS8rw4E8gTQLmVhjy3bDBhkkSd2VH62/Hzb7ifQcDhD4RlCutVaq
05GPHpHrqqo6C757BQcDBDRmjZhdsIIRMnTjPKk+CsRtskdzo/3Jb6Phb/XlrbOo
bpMvCpnNWPLbh4CNuxzRdTpcp0syuGEcuE8z6tpLWy2SFrMsKiO6xCcEgzF/z6i/
fmMm1AYzCNvQHy9UbjDYposQwzUlMRZhb5uh046HHHMwn/zTm9ZDH7Tz4L5VptkQ
4GxfVWAOtJ5W1pfglrYvOjL7IyD0BDqIjT8CXQOu0u+TcJfW+x9WOLw9c+3uHt7q
ySAc7clYOTiKZgddi0tCIiWwzWgRwVwlo0a6atJMYO3I5UWME2zvByADKu5tRZcE
qxL85moAZT8Wtt1wGHWWY2fiChIkPVWEtB5QljtSv3rqDmktsprax9OTlwZaHyQ4
3wibik1QNKrPql79GQRsRZGKAcfRA3xzKE1Uefydxje+Oh/v6pU3TbAhmwouwT7f
F7ud9U7qdVU7dqgfAPzy+igGY5vrNl7WJj9q7Yrau2Psb2bByUsf2UUE8PbO857s
O6g7Mfkqn4y8eSzg3KKSTOV5PTG8PdwVPz2qExzoRRddoIZW9zcu7P+qNQEsLyWt
Ev4Aa43fEbNUKr4cvh0XeZZ4VVoavLgoiQGcBBABAgAGBQJV+wVdAAoJEIi9mb9G
u5s9OW0MAIbI1RwqYWAvZZc+So/x1LOSYRRxNViBbSQG4FIZ9U6uZz/2m8eo21nz
FyIkig/EfkDKOkirjfwliuReNvcUpj4AKdnM48YlYajJjktvT67GPXhR9KZobo4M
21azuPhF/ynbfzigNJUWWxNqweS0oLac3YvQS89qawblEycCo2pvTE8ngNWsTjRQ
u3BwxJkl82hih3Zc3nDHv11pSUTtOm7+xLHmR9wHa8cuNOposqPp2BKMeRbjqGyL
IAj7pSg7obXAcCdDjVQ88cNkPDPOfc4t8EcbMUUJ00vuLltwj565iZzw/REszg3l
4O+Lo8t8BtuVJo3Ey5+1attcx1bAakcNCiR+r0d8BEK0Fsh/xz27uJDXzTim2jZ8
Zc+xLQ4Cje+l7eytOZTISe/dhU/aiD1DPO5qW0q39Sok0+jxEf3ZE/R8RH1DnSEK
OG7WpDUGyS45jzvX6KOOS3vMx3x7kq8u3b2f8dI5EQrMRgMZk6u466RvVDqxw7Hh
Z6IscxUgULkCDQROnOmXARAArN10N/Yex+bUeDld7WP54R+aGzVtPeLNSToHj955
2f1kmCS9US6xZSDDUmlpAcw2QeWFUbZs8m9pwPL0EJRkH6I8g8c4+24Mjqin81Tp
j/D5Oa8VxEMq27J19gAStQX1KAB0AMaKN6JAg4qAhqY9SMzdUzj05gwZ71RUVNEC
T1fi1do2c8qaZIsVR9ZhEkY8ZPDynHc0WSsJZqPophe3gyLKXqZK+9SiSJeosGW8
QHGxWcR7oOT0PWdF5dnW6IMfio6MQ08wX/uw1oyAiXw5Z0l8ZggDpfIDkgFeb5uw
G9BNtV0B5Rk4rbIFRk+QfiHtvo3NWfBunMILIJfN9mJvlnLrQI2Dl6kqQchuqj0A
mkhc1OGPQWfhVzW3TCFWOwbIg5zs/5spa1tQ4xB2Iug6tlPD5r8AzgIVxe4EEXs6
qYQ02g17unSO1YDmc2DdjsN44iSLprPTBNgNBS82IQzMiifoFjoAOdZs+0l+b5++
cbCTdB+nDZqOTeIH15A5CJwoFbpBlcdJoz55RMAt/3j74q5mx7BYensFKvaz50w/
ItZOnrOgoXMb2e1sl0BUv523JnKRtwEqYDtg8Ti/0Jy3u7LN4oJupzV7K2zGRyXP
K7zxHgziN1hIpxnb6FJRqUUDPQ47DNE7Pj+Jd0jxdhOUvSt7BGbSPJGAQ1Ob9fR+
67UAEQEAAYkCHwQYAQIACQUCTpzplwIbDAAKCRDZIA5s0a248UUBD/0e4WMJjEwF
ftCFxciVvmv8YyM32DFwXLsiyxMWNmF/XMZkXhr6SSh9zh1C5MUAkf3PIfzGa66/
tKIvuhwRiXCQC8F8/ZFuzp99Hh2VgUUSju2OWPoisxJAYUrGaHibk9b1tOQu2LaW
WSHsmwku3PT0hPaHYIHnxMSdH8jpVFDxoaOe0RsTXa+d0ptjYAXl0IKKlx/g/nIs
cdqo59jfc5pD/n7Afl6r3A7rxOPEHZ1XhSiazfKx6fmIyQtynnxRJLKHu0GT4evc
R6wbTze2pflBBSp4ojw/kSIILT+724dxnlA/7SLhokYRivvJ36EMCrG2H8JqAAbZ
Bl1SHT/t7jGFXLRAEMhIHXHtv831zjD4lwUbB7l3rHUCH0RO9PvIn3R1itHUmbqU
++mQlNe6tX5avd3Yf1XUHHS/MR4eHav10QdCDSbFYZMGTWVkfJuebfgM32Uo/fDy
Z4FtdujvjvuIgYF6dlP23sIbX7B+JHxo21sgkghmVh3A0xyhKEWbZl0ADDEJwN8P
ugRTyHv1ewKJcILwTc+o7J3Nl9uOAVTS6vmVkLoGsxk+tsJEWbpLtWthb3i2rtZ3
uJOCwNU7Ox85KZM6PBtG5Ejca//fTd3db1IXtlEoP/e/U+4aQznVUtGh4KrZeCBw
Gi4h6MYZYulcdIXW0RoPSFPsI267MfsG/Q==
=rRox
-----END PGP PUBLIC KEY BLOCK-----
```

**Comment by rustyrussell** [comment]
Thanks, that tells me what's happening.  hsmd is getting hammered by requests, writing out its status and filling the pipe.  Since lightningd isn't reading the status, but is  waiting on the hsm reply, it's stuck.

If I'm right, you should be able to trivially fix this by commenting out or deleting line 820 of hsmd/hsm.c:

```
	status_debug("Client: Received message %d from client", t);
```

**Comment by robtex** [comment]
I remarked that line and it seems it started rolling and scrolling
@rustyrussell you take "writing remarkable code" to a totally unexpected level!

**Comment by robtex** [comment]
fyi It worked a good while, but i noticed it stopped.
i just started it again and it's rolling. not synced up yet, 3000 blocks to go.

```
Program received signal SIGPIPE, Broken pipe.
0x00007ffff73102c0 in __write_nocancel () at ../sysdeps/unix/syscall-template.S:84
84      ../sysdeps/unix/syscall-template.S: No such file or directory.                                                                                                                                              (gdb) bt
#0  0x00007ffff73102c0 in __write_nocancel () at ../sysdeps/unix/syscall-template.S:84
#1  0x000000000043f88d in do_write_wire_header (fd=95, arg=0x34c9818) at wire/wire_io.c:92
#2  0x000000000043f91e in do_write_wire (fd=95, arg=0x34c9818) at wire/wire_io.c:111
#3  0x0000000000466925 in do_plan (conn=0x34c9798, plan=0x34c97f8, idle_on_epipe=true) at ccan/ccan/io/io.c:374
#4  0x0000000000466a28 in io_ready (conn=0x34c9798, pollflags=21) at ccan/ccan/io/io.c:403
#5  0x0000000000468537 in io_loop (timers=0x71d228, expired=0x7fffffffde58) at ccan/ccan/io/poll.c:310
#6  0x0000000000414b1f in main (argc=1, argv=0x7fffffffdf88) at lightningd/lightningd.c:455
```

**Comment by rustyrussell** [comment]
it could take quite a while to catch up from that far behind....

**Comment by robtex** [comment]
it is caught up now and seems to run a while, then it borks
```
Program received signal SIGPIPE, Broken pipe.
0x00007ffff73102c0 in __write_nocancel () at ../sysdeps/unix/syscall-template.S:84
84      ../sysdeps/unix/syscall-template.S: No such file or directory.
(gdb) bt
#0  0x00007ffff73102c0 in __write_nocancel () at ../sysdeps/unix/syscall-template.S:84
#1  0x000000000043f88d in do_write_wire_header (fd=102, arg=0x131cc28) at wire/wire_io.c:92
#2  0x000000000043f91e in do_write_wire (fd=102, arg=0x131cc28) at wire/wire_io.c:111
#3  0x0000000000466925 in do_plan (conn=0x131cba8, plan=0x131cc08, idle_on_epipe=true) at ccan/ccan/io/io.c:374
#4  0x0000000000466a28 in io_ready (conn=0x131cba8, pollflags=21) at ccan/ccan/io/io.c:403
#5  0x0000000000468537 in io_loop (timers=0x71d228, expired=0x7fffffffde58) at ccan/ccan/io/poll.c:310
#6  0x0000000000414b1f in main (argc=1, argv=0x7fffffffdf88) at lightningd/lightningd.c:455
```

and last in log is
```
2018-08-16T22:55:09.954Z lightning_gossipd(22851): Received channel_update for 531367:2138:1 with old time 1531591791
2018-08-16T22:55:09.955Z lightningd(22841): lightning_channeld-032e04b67641c00444af1d83145c0b63bac8316a6afb8fec0f87938295ed8bb129 chan #1115: UPDATE WIRE_CHANNEL_GOT_REVOKE
2018-08-16T22:55:09.955Z lightningd(22841): 032e04b67641c00444af1d83145c0b63bac8316a6afb8fec0f87938295ed8bb129 chan #1115: got revoke 179: 0 changed
2018-08-16T22:55:09.994Z lightningd(22841): lightning_channeld-02a13a67131bb41c8d6b858d8c2f37b597e07a2341b9e393ac1cd3f7e9cefc4760 chan #1089: Can't send commit: nothing to send
2018-08-16T22:55:09.994Z lightningd(22841): lightning_channeld-037b80d06df0e9d764b1e2626bc5c6a6e79b6da32f28e65815b1d91436d6dd9792 chan #1041: Received revoke_and_ack
2018-08-16T22:55:09.994Z lightningd(22841): lightning_channeld-02279affa59de742503180ee0745be70f827f923ddf85ac6f811e5f629f6966f34 chan #1048: Sending master 1022
2018-08-16T22:55:09.994Z lightning_gossipd(22851): Received channel_update for 531784:875:0 with old time 1532365058
2018-08-16T22:55:09.994Z lightningd(22841): lightning_channeld-032e04b67641c00444af1d83145c0b63bac8316a6afb8fec0f87938295ed8bb129 chan #1115: ... , awaiting 1122
2018-08-16T22:55:09.994Z lightningd(22841): lightning_channeld-037b80d06df0e9d764b1e2626bc5c6a6e79b6da32f28e65815b1d91436d6dd9792 chan #1041: No commits outstanding after recv revoke_and_ack
2018-08-16T22:55:09.994Z lightningd(22841): lightning_channeld-02279affa59de742503180ee0745be70f827f923ddf85ac6f811e5f629f6966f34 chan #1048: UPDATE WIRE_CHANNEL_GOT_REVOKE
2018-08-16T22:55:09.995Z lightningd(22841): 02279affa59de742503180ee0745be70f827f923ddf85ac6f811e5f629f6966f34 chan #1048: got revoke 126: 0 changed
2018-08-16T22:55:10.015Z lightningd(22841): lightning_channeld-033cfe09030dff97391e08cd08c77834efcb44b706bee7ac7d0866b02ed67f356a chan #870: peer_in WIRE_ERROR
2018-08-16T22:55:10.015Z lightningd(22841): Feerate estimate for Immediate set to 2967 (was 2966)
2018-08-16T22:55:10.015Z lightningd(22841): Feerate estimate for Normal set to 2437 (was 2436)
2018-08-16T22:55:10.015Z lightningd(22841): ... feerate estimate for Slow hit floor 253
```
is it related or shall we make a new issue?

**Comment by rustyrussell** [comment]
That's something else... fd 105 is not the hsm, it's another subdaemon.  We should never block here; if you type 'return' in the debugger does it hang?

**Comment by robtex** [comment]
tried return fyi... saw you fixed something. i'll try updating to latest master now.

```Starting program: /usr/local/bin/lightningd
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".

Program received signal SIGPIPE, Broken pipe.
0x00007ffff73102c0 in __write_nocancel () at ../sysdeps/unix/syscall-template.S:84
84      ../sysdeps/unix/syscall-template.S: No such file or directory.
(gdb) return
Make selected stack frame return now? (y or n)
Please answer y or n.
Make selected stack frame return now? (y or n) y
#0  0x000000000043f88d in do_write_wire_header (fd=82, arg=0x66c4c18) at wire/wire_io.c:92
92              ret = write(fd, (char *)&hdr + len, HEADER_LEN - len);
(gdb) return
Make do_write_wire_header return now? (y or n) y
#0  0x000000000043f91e in do_write_wire (fd=82, arg=0x66c4c18) at wire/wire_io.c:111
111                     return do_write_wire_header(fd, arg);
(gdb) return
Make do_write_wire return now? (y or n) y
#0  0x0000000000466925 in do_plan (conn=0x66c4b98, plan=0x66c4bf8, idle_on_epipe=true) at ccan/ccan/io/io.c:374
374             switch (plan->io(conn->fd.fd, &plan->arg)) {
```

**Comment by robtex** [comment]
@cdecker it looks quite similar on latest master

```
(gdb) r
Starting program: /usr/local/bin/lightningd
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".

Program received signal SIGPIPE, Broken pipe.
0x00007ffff73102c0 in __write_nocancel () at ../sysdeps/unix/syscall-template.S:84
84      ../sysdeps/unix/syscall-template.S: No such file or directory.
(gdb) bt
#0  0x00007ffff73102c0 in __write_nocancel () at ../sysdeps/unix/syscall-template.S:84
#1  0x000000000044033e in do_write_wire_header (fd=80, arg=0x1c19b58) at wire/wire_io.c:92
#2  0x00000000004403cf in do_write_wire (fd=80, arg=0x1c19b58) at wire/wire_io.c:111
#3  0x0000000000467afe in do_plan (conn=0x1c19ad8, plan=0x1c19b38, idle_on_epipe=true) at ccan/ccan/io/io.c:374
#4  0x0000000000467c01 in io_ready (conn=0x1c19ad8, pollflags=21) at ccan/ccan/io/io.c:403
#5  0x0000000000469710 in io_loop (timers=0x71f228, expired=0x7fffffffde58) at ccan/ccan/io/poll.c:310
#6  0x00000000004154ef in main (argc=1, argv=0x7fffffffdf88) at lightningd/lightningd.c:455
```

**Comment by cdecker** [comment]
Ok, will reopen as it is not fixed yet.

**Comment by rustyrussell** [comment]
Closing this, since we've not had any further reports...
